db-connect.php

```diff
@@ -1,10 +1,20 @@
 <?php 
```

Comment:
Is there any reason that `db-connect.php` was moved out to the `templates` folder?

Reply:
Yes , I tried using the composer inside the template folded so , I moved out the file .

Reply:
Did you test the following pages after this?
- login/signup
- data -> summary
- data -> overview
- search (enter some pair id)
- global match
- pairwise match

Reply:
Since the functionality of most of the pages depends on this file.

Reply:
Ya I tested all of them

Reply:
Awesome!